website/source/docs/guides/prod.html.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: "docs"
+page_title: "Production Hardening"
+sidebar_current: "docs-guides-prod"
+description: |-
+  Generate a new root token using a threshold of unseal keys.
+---
+
+# Generate Root Tokens Using Unseal Keys
+
+It is generally considered a best practice to not persist
+[root tokens][root-tokens]. Instead a root token should be generated using
+Vault's `generate-root` command only when absolutely necessary. This guide
+demonstrates regenerating a root token.
+
+1. Unseal the vault using the existing quorum of unseal keys. You do not need to
+  be authenticated.
+
+    ```shell
+    $ vault unseal
+    # ...
+    ```
+
+2. Generate a one-time password:
+
+    ```shell
+    $ vault generate-root -genotp
+    ```
+
+3. Get the encoded root token:
+
+    ```shell
+    $ vault generate-root -otp="<otp>"
```

Comment:
The `-otp=` flag doesn't need to be passed on every generate-root call while trying to make quorum. That way the operator who happens to enter their unseal key last needs to coordinate with the operator who initialized the generate root in order to decrypt the root token.